lib/path_processing.py

```diff
@@ -25,28 +27,33 @@ def remove_com_vmware_from_dict(self, swagger_obj, depth=0, keys_list=[]):
             for key, item in swagger_obj.items():
                 if isinstance(item, str):
                     if key in ('$ref', 'summary', 'description'):
-                        item = item.replace('com.vmware.', '')
+                        item = item.replace('com.vmware.', '').replace('ComVmware', '')
```

Comment:
is sounds to me that this removal happens too late in the game, given that we have to handle both formats here (and below in this function), the source was certainly in single format. 

But anyways, this is probably a bigger refactoring, which might not be worth it right now, so feel free to resolve this w/out any action.

Reply:
No action taken, since resolution will require a lot of refactoring